src/lib/components/workflow-actions.svelte

```diff
@@ -80,7 +80,7 @@
 
   const handleTerminationError = (error: NetworkError) => {
     reason = '';
-    terminateConfirmationModal.setError(
+    terminateConfirmationModal?.setError(
```

Comment:
Thoughts on using a pattern of only setting an error on the modal if it exists, otherwise adding a notification?